PerspectiveRailroad/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      language="javascript"
+      type="text/javascript"
+      src="../libraries/p5.min.js"
+    ></script>
+    <script type="module">
+      import { sketch } from "./PerspectiveRailroad.js";
+      import "../common/sketchbook-navbar.js";
+      const p5_sketch = new p5(sketch, "sketch");
+    </script>
+    <link rel="stylesheet" href="../styles/layout.css" />
+    <style>
+      img {
+        display: block;
+        margin: 0 auto;
+      }
+    </style>
+  </head>
+  <body>
+    <sketchbook-navbar></sketchbook-navbar>
+    <div class="container">
+      <h1>Perspective Railroad (2025)</h1>
+      <div id="sketch"></div>
+      <div class="description">
+        <p>
+          This animation draws inspiration from both math and art. I recently
+          read
+          <a
+            href="https://www.google.com/books/edition/_/YQ3jDAAAQBAJ?hl=en&kptab=overview"
+            >Perspective Made Easy</a
+          >
+          by Ernest R. Norling. The book is a handy tutorial on drawing in
+          perspective in traditional art.
```

Comment:
Mention the fencepost method and dividing a perspective rectangle into even-width subdivisions being relevant here.